manifests/params.pp

```diff
@@ -5,6 +5,7 @@
       case $::operatingsystem {
```

Comment:
You could add the variable assignment above this line, similar to the Debian case below.